site/layouts/shortcodes/helm-version.html

```diff
@@ -0,0 +1,7 @@
+{{- $pagePrefix := (index (split $.Page.File.Dir "/") 0) -}}
+{{- with (eq $pagePrefix "latest") -}}
+{{- "v0.0.0-latest" -}}
+{{- end -}}
+{{- with (strings.HasPrefix $pagePrefix "v1.1") -}}
+{{- "v1.1.0" -}}
```

Comment:
need update this when releasing a minor patch.

Reply:
Do we also need to update the release guide?

Reply:
Absolutely, there were a lot of mistakes on these things.